runtime/runtime-core/native/src/aws/smithy/kotlin/runtime/io/internal/SdkDispatchersNative.kt

```diff
@@ -6,12 +6,14 @@ package aws.smithy.kotlin.runtime.io.internal
 
 import aws.smithy.kotlin.runtime.InternalApi
 import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.IO
 
 @InternalApi
 public actual object SdkDispatchers {
     /**
      * The CoroutineDispatcher that is designed for offloading blocking IO tasks to a shared pool of threads.
      */
     public actual val IO: CoroutineDispatcher
-        get() = TODO("Not yet implemented")
+        get() = Dispatchers.IO
 }
```

Comment:
note: we made this `expect/actual` because Native didn't have Dispatchers.IO at the time. It does now, so we _could_ remove SdkDispatchers... but I think we'll need it again in the future when we support platforms like JS and WASM.

Reply:
Unless JetBrains implements dispatchers on those platforms first! 😁